src/components/pages/Offer/StocksManager/StockItem/decorators/__specs__/adaptBookingLimitDatetimeGivenBeginningDatetime.spec.js

```diff
@@ -71,13 +70,32 @@ describe('src | components | pages | Offer | StockManager | StockItem | decorato
         const result = updateBookingLimitDatetime({
           beginningDatetime,
           bookingLimitDatetime,
-          isEvent
+          isEvent,
         })
 
         // then
         expect(result.bookingLimitDatetime).toStrictEqual('2019-04-20T19:00:00.000Z')
       })
     })
+
+    describe('when bookingLimiteDateTime is empty', () => {
+      it('should leave bookingLimitDatetime empty', () => {
```

Comment:
```suggestion
      it('should leave the booking limit date time empty', () => {
```
L'intitulé d'un test ne doit pas être technique mais refléter le fonctionnel et donc être lisible par n'importe qui.